dvc/api.py

```diff
@@ -27,7 +27,8 @@ def get_url(path, repo=None, rev=None, remote=None):
             raise OutputNotFoundError(path, repo)
 
         cloud = info["repo"].cloud
-        md5 = info["repo"].dvcfs.info(fs_path)["md5"]
+        dvc_path = _repo.fs.path.relpath(fs_path, info["repo"].root_dir)
```

Comment:
Will go away after the switch in repofs. Also should use dvcfs with subrepo support here, which will be done in the future (have similar requests in ls and diff functionality).